src/inject/GPMInject/interface/toast.js

```diff
@@ -0,0 +1,27 @@
+import { remote } from 'electron';
+
+Emitter.on('pauseAfter:show', () => {
+  const toast = document.createElement('paper-toast');
```

Comment:
Hmm, it would have to be a method with loads of parameters... There are very few things absolutely the same...